### PR TITLE
Move config/middlewares to simply config.go on middlewares

### PR DESCRIPTION
**Pull request by lerrua**
Hi guys,

I'm trying to solve that issue https://github.com/prest/prest/issues/207, bringring back config/middlewares file to middlewares repository as `middlewares/config.go`. 
It will avoid developers to import two middlewares packages.

### CODE REVIEW

**Review by crgimenes — APPROVED** [review]
LGTM

**Review by felipeweb — APPROVED** [review]
LGTM

File: config_test.go [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+
+func init() {
+	config.Load()
+}
+
+func TestInitApp(t *testing.T) {
+	app = nil
+	initApp()
+	if app == nil {
+		t.Errorf("App should not be nil.")
```
Comment (marioidival) [inline config_test.go:27]: Error strings should not be capitalized

**Review by marioidival — COMMENTED** [review]
Man, Error strings should not be capitalized

**Pull request merged**